scripts/templates/ldrddi.cpp.mako

```diff
@@ -82,7 +82,13 @@ namespace loader
             uint32_t library_driver_handle_count = 0;
 
             result = drv.dditable.${n}.${th.get_table_name(n, tags, obj)}.${th.make_pfn_name(n, tags, obj)}( &library_driver_handle_count, nullptr );
-            if( ${X}_RESULT_SUCCESS != result ) break;
+            if( ${X}_RESULT_SUCCESS != result ) {
```

Comment:
Will it never be the case that a driver init status could change from Uninitialized after the first call?

Reply:
The init status will be updated on the call to zeInit for that driver, but that will be before the call to driverget.

https://github.com/oneapi-src/level-zero/blob/master/source/loader/ze_ldrddi.cpp#L52

The only time the init status may change after a call is if we are in instrumentation callback, but then the instrumentation excluded driver types would be the one that failed.

Therefore, the call to driver get from the instrumentation callback will only get the drivers that the init flag allowed, but the later call to init that allows all drivers would get the full set of drivers because the init flag was updated.